src/app/body/common-tools/suggestion-bucket/suggestion-bucket.component.ts

```diff
@@ -26,6 +26,8 @@ export class SuggestionBucketComponent implements OnInit {
 
   @Input("email") email: string;
   @Input("showOnlyProfilePic") showOnlyProfilePic: boolean = false;
+
+  @Input("showEmail") showEmail: boolean = false;
```

Comment:
You will have to receive this value from all the parents calling this component

Reply:
Done